dotcom-rendering/src/components/MainMediaGallery.tsx

```diff
@@ -36,16 +46,40 @@ export const MainMediaGallery = ({ mainMedia, format }: Props) => {
 
 	return (
 		<figure css={styles}>
-			<Picture
-				alt={mainMedia.data.alt ?? ''}
-				format={format}
-				role={mainMedia.role}
-				master={asset.url}
-				width={width}
-				height={height}
-				loading="eager"
-				isMainMedia={true}
-			/>
+			{renderingTarget === 'Apps' ? (
+				<Island priority="critical">
+					<AppsLightboxImage
+						elementId={mainMedia.elementId}
+						role={mainMedia.role}
+						format={format}
+						master={asset.url}
+						alt={mainMedia.data.alt ?? ''}
+						width={width}
+						height={height}
+						loading={'lazy'}
+					/>
+				</Island>
+			) : (
+				<Picture
+					alt={mainMedia.data.alt ?? ''}
+					format={format}
+					role={mainMedia.role}
+					master={asset.url}
+					width={width}
+					height={height}
+					loading="eager"
+					isMainMedia={true}
+				/>
+			)}
+			{renderingTarget === 'Web' && !isUndefined(mainMedia.position) && (
+				<LightboxLink
+					role={mainMedia.role}
+					format={format}
+					elementId={mainMedia.elementId}
+					isMainMedia={false}
```

Comment:
I think this is the main media?

```suggestion
					isMainMedia={true}
```

It appears that this is currently used to hide the lightbox overlay for immersives, so perhaps we want to update `LightboxLink` to introduce this behaviour for galleries too?

https://github.com/guardian/dotcom-rendering/blob/c2de706b61d2c8e32e7727ab8bb3fc2aecbac68d/dotcom-rendering/src/components/LightboxLink.tsx#L153-L155